src/tiled/commanddialog.cpp

```diff
@@ -63,6 +69,24 @@ void CommandDialog::closeEvent(QCloseEvent *event)
     CommandManager::instance()->updateActions();
 }
 
+void CommandDialog::setShortcut(const QKeySequence &keySequence)
+{
+    if (!mUi->keySequenceEdit->isEnabled())
+        mUi->keySequenceEdit->setEnabled(true);
```

Comment:
A public slot is just a public method that can optionally be used in an old-style connect statement (since it can be dynamically called by its name, which also makes it callable from JavaScript for example). There is no reason not to also use it at a plain function. In fact, many of the Qt APIs are slots even though they are mostly called directly, just because it enables the convenience of connecting things sometimes.